src/main/java/yahav/photomosaic/ImagePixels.java

```diff
@@ -0,0 +1,72 @@
+package yahav.photomosaic;
+
+import javafx.geometry.Rectangle2D;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.image.PixelFormat;
+import javafx.scene.image.PixelReader;
+import javafx.scene.paint.Color;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ImagePixels{
+
+    ArrayList<Rectangle2D> squareInput = new ArrayList<>(); //array of input squares to compare to source images
+    public ImagePixels() throws IOException {
+        FileInputStream input = new FileInputStream("src/main/resources/google.png");
+        Image image = new Image(input);
+        ImageView imageView = new ImageView();
+        PixelReader pixelReader = image.getPixelReader();
+    }
+
+
+    public Color getImagePixels(Image image) {
```

Comment:
This method should be called something like `getAverageColor` since `getImagePixels` doesn't describe what it does.